core/rawdb/accessors_skipped_txs.go

```diff
@@ -51,6 +52,12 @@ type SkippedTransaction struct {
 	// We store the tx itself because otherwise geth will discard it after skipping.
 	Tx *types.Transaction
 
+	// Traces is the wrapped traces of the skipped transaction.
+	// We only store it when `MinerStoreSkippedTxTracesFlag` is enabled, so it might be empty.
+	// Note that we don't use *types.BlockTrace directly because types.BlockTrace.StorageTrace.Proofs is of
+	// type map[string][]hexutil.Bytes, and is not RLP-serializable.
+	TracesBytes []byte
```

Comment:
This will break `readSkippedTransactionRLP`, it will fail on transactions stored before this upgrade. It's better to add a `SkippedTransactionV2` and try decoding as `SkippedTransactionV2` first, then as `SkippedTransaction`.

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/500/commits/ae0dca4353a5d0d147131fcb083b3c766bb82241